### ISSUE TITLE
Support new XR Plugin Manager

### ISSUE THREAD

**Issue by haywirephoenix** [Unity-Technologies/EditorXR#566]
The new XR Plugin management is replacing the old method of importing a vr platfrom package.

I thought  #548 on 0.3.0 meant that it would now be compatible with this new universal xr system?

It seems it cant detect the new oculus plugin and is only looking for the soon deprecated "virtual reality supported". Thus leaving EditoXR greyed out. 

Changing PlayerSettings.virtualRealitySupported to true gets me the window but no hardware is detected in  the viewer

**Comment by haywirephoenix** [comment]
![Capture](https://user-images.githubusercontent.com/26271795/73195408-b9802580-4125-11ea-89af-10735e00baa5.PNG)

**Comment by amirebrahimi** [comment]
Yes, this is currently unsupported and we'll need to update it soon. For now, you'll need to use the Oculus Desktop or OpenVR Desktop packages from the Package Manager.

**Comment by haywirephoenix** [comment]
Okay thank you for that. How about 2019.3 compatibility? I have a functioning exr window now but input is ignored. Do I need some kind of steam input profile?

**Comment by Par3val** [comment]
Checking in to see if any progress has been made?

**Comment by amirebrahimi** [comment]
Unfortunately, no significant news to report; We have a task force that is looking into what's needed for updating it, so we can resource it properly.

**Comment by danilofaugusto** [comment]
Folks from Unity, the EditorXR is amazing! Kudos to you for the great work you are doing.

I hope the XR Manager support is coming soon, I would love to create the levels for my VR experiences with this tool.

**Comment by MileyHollenberg** [comment]
Has there been any update on this? The latest version on the Package Manager at least doesn't seem to support the new XR Manager

**Comment by amirebrahimi** [comment]
"Coming soon" :tm:

**Comment by valentinwinkelmann** [comment]
It's very sad that no further progress can be seen here. Bump for that..

**Comment by haywirephoenix** [comment]
Guys, the EditorXR page is still up at [https://unity.com/editorxr](https://unity.com/editorxr) check it out to remind you why it's awesome. 

P.S: I installed UE5 and they're still supporting their VR Editor plugin.
